extensions/io/org.eclipse.smarthome.io.iota/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>pom</artifactId>
+    <groupId>org.eclipse.smarthome.io</groupId>
+    <version>0.10.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.eclipse.smarthome.io.iota</artifactId>
+  <packaging>eclipse-plugin</packaging>
+
+  <name>Eclipse SmartHome Iota IO</name>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.8</version>
+        <executions>
+          <execution>
+            <phase>initialize</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <delete 
+                  dir="lib/mam"
+                  includeemptydirs="true"/>
+              </target>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>1.3.2</version>
+        <executions>
+            <execution>
+                <id>github-mam-clone</id>
+                <goals>
+                    <goal>exec</goal>
+                </goals>
+                <phase>generate-resources</phase>
+                <configuration>
+                    <executable>git</executable>
+                    <arguments>
+                        <argument>clone</argument>
+                        <argument>https://github.com/GiovannaTheo/mam.client.js.git</argument>
```

Comment:
I did not change anything regarding the core functions of MAM. My repo only is a clone of the official repo, but with custom scripts in the example folder, made in such a way that they can be invoked through command line. I'm guessing since we're having issues with Licensing, it is better to left it this way atm and see in the future what happens.

Cutting off MAM support would basically render the project useless since both client and IO bundles need MAM to work.